bigquery_etl/view/__init__.py

```diff
@@ -282,12 +282,18 @@ def has_changes(self, target_project=None):
             print(f"view {target_view_id} will change: does not exist in BigQuery")
             return True
 
-        expected_view_query = CREATE_VIEW_PATTERN.sub(
-            "", sqlparse.format(self.content, strip_comments=True), count=1
-        ).strip(";" + string.whitespace)
-        actual_view_query = sqlparse.format(
-            table.view_query, strip_comments=True
-        ).strip(";" + string.whitespace)
+        try:
+            expected_view_query = CREATE_VIEW_PATTERN.sub(
+                "", sqlparse.format(self.content, strip_comments=True), count=1
+            ).strip(";" + string.whitespace)
+
+            actual_view_query = sqlparse.format(
+                table.view_query, strip_comments=True
+            ).strip(";" + string.whitespace)
+        except TypeError as err:
+            print(f"ERROR: There has been an issue formating: {target_view_id}")
+            raise err
```

Comment:
(In case you didn't know) you can just `raise` to do a reraise.  Doesn't matter though
```suggestion
            raise
```

Reply:
ah ok, so it will just automatically use the most recent exception?

Reply:
Yes whatever it's currently "excepting"